components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -200,39 +257,50 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
         return withKeys(separate(renderableInfoParts, () => getSeparator()));
     }, [info, getSeparator]);
 
-    const getInfloListItemContent = (
+    const getInfoListItemContent = (): JSX.Element => (
         <>
-            <div className={combine('statusStripe')} data-test={'status-stripe'} />
-            {divider && <Divider className={combine('divider')} />}
+            <StatusStripe statusColor={statusColor} className={combine('statusStripe')} data-test={'status-stripe'} />
+            {divider && <InfoListItemDivider divider={divider} className={combine('divider')} />}
             {(icon || !hidePadding) && getIcon()}
             {leftComponent}
-            <ListItemText
+            <InfoListItemText
                 primary={title}
+                leftComponent={leftComponent}
                 className={combine('listItemText')}
                 secondary={
                     subtitle || info ? (
                         <>
                             {subtitle ? (
-                                <Typography
-                                    variant={'subtitle2'}
-                                    component={'p'}
+                                <Subtitle
+                                    variant="subtitle2"
+                                    component="p"
+                                    fontColor={fontColor}
                                     noWrap={!wrapSubtitle}
                                     className={combine('subtitle')}
                                 >
                                     {getSubtitle()}
-                                </Typography>
+                                </Subtitle>
                             ) : undefined}
                             {info ? (
-                                <Typography variant={'body2'} noWrap={!wrapInfo} className={combine('info')}>
+                                <Info
+                                    variant={'body2'}
+                                    fontColor={fontColor}
+                                    noWrap={!wrapInfo}
+                                    className={combine('info')}
+                                >
                                     {getInfo()}
-                                </Typography>
+                                </Info>
                             ) : undefined}
                         </>
                     ) : undefined
                 }
                 primaryTypographyProps={{
                     noWrap: !wrapTitle,
                     variant: 'body1',
+                    fontWeight: 600,
+                    lineHeight: 1.25,
+                    display: 'block',
+                    color: fontColor || 'inherit',
```

Comment:
What's the motive for specifying these values through props instead of CSS?

Reply:
Who's motive?! I don't even know who did this work at this stage in the game. 

gitblame suggests @bkarambe about 2 months ago ¯\_(ツ)_/¯ 

@bkarambe any thoughts on this?

Reply:
I mean, it's fine. I'm just curious.

Reply:
> Who's motive?! I don't even know who did this work at this stage in the game.
> 
> gitblame suggests @bkarambe about 2 months ago ¯_(ツ)_/¯
> 
> @bkarambe any thoughts on this?

primaryTypographyProps was there previously, so just added the other styles. We can clean this up to pass through css.

Reply:
Leaving this as is since it's working right now. We can revisit this later if we think we can optimize this a bit by moving the style declaration to styles instead of using the typography props